modules/cli-installing-cli.adoc

```diff
@@ -38,25 +38,93 @@ of the commands in {product-title} {product-version}. Download and
 install the new version of `oc`.
 ====
 
+[id="cli-installing-cli-on-linux_{context}"]
+== Installing the CLI on Linux
+
+You can install the OpenShift CLI (`oc`) binary on Linux using the following procedure.
+
 .Procedure
+
 ifdef::openshift-origin[]
-. Navigate to https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ and choose the folder for your operating system
-. Download `oc.tar.gz`
+. Navigate to https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ and choose the folder for your operating system and architecture.
+. Download `oc.tar.gz`.
 endif::[]
 ifndef::openshift-origin[]
-. From the link:https://cloud.redhat.com/openshift/install[Infrastructure Provider] page on the {cloud-redhat-com} site, navigate to the page for your installation type and
-click *Download Command-line Tools*.
+. From the link:https://cloud.redhat.com/openshift/install[Infrastructure Provider] page on the {cloud-redhat-com} site, navigate to the page for your installation type.
+. In the *Command-line interface* section, select *Linux* from the drop-down menu and click *Download command-line tools*.
 endif::[]
-. Click the folder for your operating system and architecture and click the
-compressed file.
+. Unpack the archive:
 +
-[NOTE]
-====
-You can install `oc` on Linux, Windows, or macOS.
-====
-. Save the file to your file system.
-. Extract the compressed file.
-. Place it in a directory that is on your `PATH`.
+----
+$ tar xvzf <file>
+----
+. Place the `oc` binary in a directory that is on your `PATH`.
++
+To check your `PATH`, execute the following command:
++
+----
+$ echo $PATH
+----
+
+After you install the CLI, it is available using the `oc` command:
+
+----
+$ oc <command>
+----
+
+[id="cli-installing-cli-on-windows_{context}"]
+== Installing the CLI on Windows
+
+You can install the OpenShift CLI (`oc`) binary on Windows using the following procedure.
+
+.Procedure
+
+ifdef::openshift-origin[]
+. Navigate to https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ and choose the folder for your operating system and architecture.
```

Comment:
s/Navigate to https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ and choose the/Navigate to the https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ page, and then choose the/